packages/amplication-data-service-generator/src/server/resource/service/create-service.ts

```diff
@@ -346,6 +357,7 @@ async function createToOneRelationFile(
 
   interpolate(toOneFile, {
     DELEGATE: delegateId,
+    PARENT_ID_TYPE: identityParentIdType(entity),
```

Comment:
How do you know which field in the entity to look at just by sending the entity to the function without the current field?

Reply:
The function is searching for the id type
```
const entityFiledId = entity.fields?.find(
    (entityField) => entityField.dataType === EnumDataType.Id
  );
```